### PR TITLE
Add object-curly-newline rule, sort rules list

### PR DESCRIPTION
**Pull request by JamesLefrere**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@colony/eslint-config-colony",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (chmanie) [inline package.json]: I think this should be done in a separate commit `npm version patch`

Comment (JamesLefrere) [reply package.json]: Agreed, let's do it on master
